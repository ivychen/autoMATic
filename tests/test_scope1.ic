int main() {
    int x;
    int y = 3;
    {
        string z = "hello!";
        print(z);
        x = 5;
    }

    print(x);
    print(y);
}
