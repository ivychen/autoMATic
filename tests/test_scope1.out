hello!
5
3
